### ISSUE TITLE
Requesting update of windows installer .exe download to current code base

### ISSUE THREAD

**Issue by Hunanbean** [morevnaproject-org/papagayo-ng#75]
Requesting update of Releases .exe download to current code base as several issues exist in the 2017 release that is listed as current release.

Thank you

**Comment by morevnaproject** [comment]
Yeah, sorry, my bad. We haven't released anything in a while... ^__^"

**Comment by Hunanbean** [comment]
No worries. I am also about to request a merge for some extended phoneme sets, an updated dictionary, and new face models to match the extended phoneme set(s). by about to, i mean hopefully within a week. i am working on the visemes right now.

**Comment by steveway** [comment]
I might hopefully have fixed some crashes, see #74 and #77 .
I'll have to do some tests before I'll push that fix.
I think it would also be best to publish a new release with this fix in place. (Provided the fix works as expected.)

**Comment by morevnaproject** [comment]
Okay. In any case, preparing an installer will take some time on my side. ^__^

**Comment by Hunanbean** [comment]
Hello again. Just here to nudge :)

**Comment by ItsMeBobO** [comment]
Will there be a windows exe for the latest code?    I donated for the windows installer which now I see is several years old.  Thank you for your work on this.  I have been using the various windows versions for several years.

**Comment by steveway** [comment]
@morevnaproject You should have access to a current Installable Windows Version I made over Patreon.
I sent you a message there, you can use that for the gumroad downloads.
And I improved the compilation and packaging scripts, if you want to package it yourself.
